### DISCUSSION TITLE
레벨4 회고

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#90] [General]
# 레벨4 회고

돌아볼까. 9시 40분 쯤에 캠퍼스에 도착해서 팀원들과 인사를 한다.
10시부터 데일리 미팅을 하면서 기분 점수를 공유한다. 난 언제나 9점 이상이었던 것 같다. 데일리 미팅이 끝나면 강의를 듣거나, 미션을 하거나, 피쳐를 개발하거나, 회의를 하거나. 항상 바빴던 기억 밖에 없다. 우테코 정규 교육 시간이 끝나고 나면 항상 오빠랑 같이 공부를 했다. 대략…

https://vero.wiki/level4-retrospect/

**Comment by somsom13** [comment]
아자자~ 게임 친구로 베로의 레벨4 회고에 한 자리 차지했다는게 이렇게 기쁠 수가~~ ( ͡° ͜ʖ ͡°)

**Reply by Cyma-s** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ my pleasure.
